### PR TITLE
Adding Morocco VAT + Fixing Bug in validate() Bulgarian VAT.

### PR DESCRIPTION
**Pull request by medmes**
Hi all,

I added Moroccan VAT Number and fixed bug in Bulgarian VAT validation.

Thanks for accepting my pull request.

Best,
Mohammed

### CODE REVIEW

**Comment by vlad-ro** [comment]
Hi Mohammed, thank you for the PR!
We'll get it merged soon.

File: src/main/java/ro/go/kpaxplanet/vat/validator/impl/BG.java [hunk 1]
```diff
@@ -68,7 +68,7 @@ public boolean validateDigits(String vatNumber) {
 		if (vatNumber.matches("^\\d\\d[0-5]\\d[0-3]\\d\\d{4}$")) {
 			// Check month
 			int month = Integer.parseInt(vatNumber.substring(2, 4));
-			if ((month > 0 && month < 13) || (month > 20 & month < 33)) {
+			if ((month > 0 && month < 13) || (month > 20 && month < 33)) {
```
Comment (cserri) [inline src/main/java/ro/go/kpaxplanet/vat/validator/impl/BG.java]: Can we add tests for this country?

Comment (medmes) [reply src/main/java/ro/go/kpaxplanet/vat/validator/impl/BG.java]: Sure , but I need to document about Bulgarian VAT number first. I think , it should be in another pull request.

Comment (cserri) [reply src/main/java/ro/go/kpaxplanet/vat/validator/impl/BG.java]: Thanks :)

**Pull request merged**